server/Models/Point.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace hutel.Models
+{
+    public class Point
+    {
+        [JsonProperty(Required = Required.Always)]
+        public Guid Id { get; set; }
+        
+        [JsonProperty(Required = Required.Always)]
+        public string TagId { get; set; }
+        
+        [JsonProperty(Required = Required.Always)]
+        public DateTime Date { get; set; }
+
+        [JsonExtensionData]
+        public Dictionary<string, object> Extra { get; set; }
```

Comment:
As described in the notes, the dictionary property with JsonExtensionData attribute is filled by json deserializer and not by caller. We don't plan on accepting complex types from outside, so no one would be able to send a dictionary.